elixir/plug/app/lib/plug_example.ex

```diff
@@ -0,0 +1,14 @@
+defmodule PlugExample do
+  import Plug.Conn
+  use Appsignal.Plug
```

Comment:
This line is triggering [an error in the build](https://appsignal.semaphoreci.com/jobs/37dd7c42-98ed-4526-acf7-a5ee29c88230#L479). Is that expected?

```
app_1    | == Compilation error in file lib/plug_example.ex ==
app_1    | ** (CompileError) lib/plug_example.ex:3: no super defined for call/2 in module PlugExample. Overridable functions available are: handle_errors/2
```